source/Tests/Playnite.Tests/Database/FiltersDatabaseTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using NUnit.Framework;
+using Playnite.Common;
+using Playnite.Database;
+using Playnite.SDK.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Playnite.Tests.Database
+{
+    [TestFixture]
+    public class FiltersDatabaseTests
+    {
+        [Test]
+        public void SortedFilterPresetsTest()
+        {
+            using (var temp = TempDirectory.Create())
+            using (var db = new TestGameDatabase(temp.TempPath))
+            {
+                db.OpenDatabase();
+                db.ClearFilterPresets();
+
+                var addedFilterPresets = new List<FilterPreset>();
+                for (int i = 0; i < 10; i++)
+                {
+                    var filterPreset = new FilterPreset() { Name = $"Preset {i}" };
+                    db.FilterPresets.Add(filterPreset);
+                    addedFilterPresets.Add(filterPreset);
+                }
+
+                // Set default sorting order based on added order
+                var newFilterPresetSettings = new FilterPresetsSettings
+                {
+                    SortingOrder = addedFilterPresets.Select(x => x.Id).ToList()
+                };
+
+                db.SetFilterPresetsSettings(newFilterPresetSettings);
+                var sortedFilterPresets = db.GetSortedFilterPresets(true);
+
+                // Verify that order is the same as when it was set
+                Assert.IsTrue(sortedFilterPresets.Select(x => x.Id).SequenceEqual(addedFilterPresets.Select(x => x.Id)));
+
+                // Set last added Filter Preset to first in order
+                newFilterPresetSettings.SortingOrder.Remove(addedFilterPresets.Last().Id);
+                newFilterPresetSettings.SortingOrder.Insert(0, addedFilterPresets.Last().Id);
+                db.SetFilterPresetsSettings(newFilterPresetSettings);
+                sortedFilterPresets = db.GetSortedFilterPresets(true);
+                Assert.AreEqual(addedFilterPresets.Last(), sortedFilterPresets.First());
+
+                // Add a new filter preset and verify that it shows as last item when obtaining sorted Filter Presets
```

Comment:
1) This needs to be tested with more than just 1 new preset
2) It needs to check that those items at the end of the list are sorted

Reply:
Done in https://github.com/JosefNemec/Playnite/pull/3258/commits/4d46394eb4383bbb9e60f82ef1798476a6dd17c4